crates/sui-core/src/authority/authority_per_epoch_store.rs

```diff
@@ -238,6 +239,7 @@ pub struct AuthorityEpochTables {
 /// Parameters of the epoch fixed at epoch start.
 #[derive(Default, Serialize, Deserialize, Debug, Eq, PartialEq)]
 pub struct EpochStartConfiguration {
+    pub epoch_id: EpochId,
```

Comment:
Why do we need this?
If you make `epoch_data` a member function of epoch store, you could populate the epoch id using the committee epoch id.

Reply:
This just makes code slightly easier (can convert EpochStartConfiguration into EpochData, otherwise we would need to pass epoch_id to the conversion function. Doable but I thought this looks better)